pyquil/quil.py

```diff
@@ -104,7 +129,8 @@ def copy_everything_except_instructions(self) -> "Program":
         new_prog = Program()
         new_prog._defined_gates = self._defined_gates.copy()
         if self.native_quil_metadata is not None:
-            new_prog.native_quil_metadata = self.native_quil_metadata.copy()
+            # TODO: remove this type: ignore once rpcq._base.Message gets type hints.
```

Comment:
```suggestion
            # TODO: remove this "type ignore" once rpcq._base.Message gets type hints.
```

Lol, mypy apparently treats any comment that contains the substring `type: ignore` as if it were a `# type: ignore` comment (fair enough, I guess).

Reply:
Amended the 9th commit.

Reply:
Actually, my previous comment is false. I just misinterpreted why @rht was seeing an error here. Mypy does not consider any comment with a `type: ignore` substring to be an ignore directive.

Reply:
OK, I have reverted to saying "type: ignore"